vars/waitForHelixRuns.groovy

```diff
@@ -52,109 +62,126 @@ def call (def helixRunsBlob, String prStatusPrefix) {
 
             // Wait until the Helix runs complete. Wait up to 5 mins between checks
             waitUntil (minRecurrencePeriod: 120, maxRecurrencePeriod: 300, unit: 'SECONDS') {
-                // Check the state against the Helix API
-                def statusUrl = "https://helix.dot.net/api/2017-04-14/jobs/${correlationId}/details"
-                def statusResponse = httpRequest statusUrl
-                assert statusResponse != null
-                assert statusResponse.content != null
-                def statusContent = (new JsonSlurperClassic()).parseText(statusResponse.content)
+                try{
+                    // Check the state against the Helix API
+                    def statusUrl = "https://helix.dot.net/api/2017-04-14/jobs/${correlationId}/details"
+                    println('Sending request to ' + statusUrl)
+                    def statusResponse = httpRequest statusUrl
+                    assert statusResponse != null
+                    assert statusResponse.content != null
+                    def statusContent = (new JsonSlurperClassic()).parseText(statusResponse.content)
+                    println('Response received from ' + statusUrl + ': ' + statusContent)
```

Comment:
You can echo the response by adding a parameter to httpRequest: consoleLogResponseBody=true

https://jenkins.io/doc/pipeline/steps/http_request/